src/components/footer/icon.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import Link from '@material-ui/core/Link';
+
+const styles = () => ({
+	icon: {
+		width: '30px'
+	}
+});
+
+class Icon extends React.Component {
```

Comment:
I suggest that we name this something else like `IconLink`, since M-UI also has a component called `Icon` and it might cause confusion later!